src/overlays/actors/ovl_En_Heishi4/z_en_heishi4.c

```diff
@@ -347,13 +347,13 @@ void EnHeishi4_Update(Actor* thisx, PlayState* play) {
     thisx->world.pos.z = this->pos.z;
     Actor_SetFocus(thisx, this->height);
     if (this->type != HEISHI4_AT_MARKET_DYING) {
-        this->unk_28C.unk_18 = player->actor.world.pos;
+        this->interactInfo.trackPos = player->actor.world.pos;
         if (!LINK_IS_ADULT) {
-            this->unk_28C.unk_18.y = (player->actor.world.pos.y - 10.0f);
+            this->interactInfo.trackPos.y = (player->actor.world.pos.y - 10.0f);
```

Comment:
```suggestion
            this->interactInfo.trackPos.y = player->actor.world.pos.y - 10.0f;
```
Not you but just something I noticed.

Reply:
also I think this change is fine to make (not sure if it was omitted on purpose, or just forgotten)